### PR TITLE
hookutils: new helper execute_multiple_root_commands

### PR DESCRIPTION
**Pull request by schopin-pro**
Very much like attach_root_command_outputs except that it simply stores the outputs in a dict rather than modifying an existing report. Very useful for post-processing.

### CODE REVIEW

File: apport/hookutils.py [hunk 1]
```diff
@@ -520,6 +523,10 @@ def attach_root_command_outputs(report, command_map):
     This is preferable to using root_command_output() multiple times, as that
     will ask for the password every time.
     """
+    if not command_map:
+        return {}
```
Comment (bdrung) [inline apport/hookutils.py:527]: Is there a useful case for that?

Comment (schopin-pro) [reply apport/hookutils.py:527]: Yes. The following pattern is fairly trivial to end up with:

```python
missing_data = {}
if not data['foo']:
    missing_data['foo'] = 'bar'
if not data['baz']:
    missing_data['baz'] = 'foo'
data.update(execute_multiple_root_commands(missing_data))
```
That way you don't need to worry about it as a caller, and having an early return means the user won't be prompted to run an empty script.

Comment (bdrung) [reply apport/hookutils.py:527]: Okay, valid point. So can we have a unit test for that?

Comment (bdrung) [reply apport/hookutils.py:527]: As mentioned on chat: The following PRs will cover this code path. So no separate test case needed.

**Review by bdrung — APPROVED** [review]

**Pull request merged**